src/www/widgets/include/interface_list.inc

```diff
@@ -1,4 +1,4 @@
 <?php
 
 $interface_list_title = gettext('Interfaces');
-$interface_list_title_link = 'status_interfaces.php';
+$interface_list_title_link = 'ui/interfaces/overview';
```

Comment:
Seeing they both point to the same page wouldn’t it make more sense to point this to the assignments page?

Reply:
Good idea :)